python/lance/tests/data/convert/test_nuscenes.py

```diff
@@ -0,0 +1,46 @@
+#  Copyright 2022 Lance Developers
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import numpy as np
+import pyarrow as pa
+import requests
+import tarfile
+
+BASE_URI_FIXTURES = "https://eto-public.s3.us-west-2.amazonaws.com"
+URI_FIXTURES = "/tests/datasets/nuimages-v1.0-mini-test-fixtures.tgz"
+LOCAL_ZIP_PATH = "nuimages-v1.0-mini-test-fixtures.tgz"
+
+from lance.data.convert.nuscenes import NuscenesConverter
+
+def download_fixtures(tmp_path):
```

Comment:
should we use `pytest.fixture` to offer a pytest style fixture?

Reply:
I feel its a bit overkill for now. If we do need more complex use of fixtures in the future, i can refactor